frontend/src/StartView.svelte

```diff
@@ -279,6 +289,13 @@
 </script>
 
 {#if !tryHash}
+  <div class="experiment-features-check">
+    <Checkbox
+      bind:checked="{$settings.experimentalFeatures}"
+      leftbox="{true}"
+      nomargin="{true}">Enable Experimental OFRAK Features</Checkbox
+    >
```

Comment:
Perhaps set `style:color="currentColor"` or something like that? Right now the other colors change when the mouse is moved, but the color of the checkbox and its label do not.